### PR TITLE
Remove unused decoder setting

### PR DESCRIPTION
**Pull request by deiteris**
Remove unused realtime decoder option and set proper operation rate.

### CODE REVIEW

**Comment by zarik5** [comment]
We are already setting operating rate and priority: https://github.com/alvr-org/ALVR/blob/master/alvr/session/src/settings.rs#L707

**Comment by deiteris** [comment]
Oh, I didn't notice. But why do we keep unused realtime decoder option then? And why operating rate is `i32::MAX`?

**Comment by zarik5** [comment]
It's not unused, we are using it here: https://github.com/alvr-org/ALVR/blob/46c7e977c22ee0ba2fe28376ac5d4f6066064c05/alvr/client_core/src/connection.rs#L315
The operating rate is set to the max value. All decoder settings are taken from the old java decoder

File: alvr/client_core/src/platform/android.rs [hunk 1]
```diff
@@ -340,6 +340,7 @@ pub fn video_decoder_split(
             format.set_str("mime", mime);
             format.set_i32("width", 512);
```
Comment (deiteris) [inline alvr/client_core/src/platform/android.rs:341]: Is it possible to somehow get foveated resolution at this point to give a proper hint to the decoder?

Comment (zarik5) [reply alvr/client_core/src/platform/android.rs:341]: Yes, we have it, but it doesn't matter, the csd0 packet already contains the resolution info.

**Comment by zarik5** [comment]
Have you tested if using the refreshrate for the operating rate improves anything?

**Comment by deiteris** [comment]
Tbh it's hard to tell if it makes any difference at all no matter if it's preferred fps, a max value or even removed completely, so I'm thinking about keeping it as is

**Comment by zarik5** [comment]
Ok, I would merge the PR as it is now. I was not in favor of routing the refresh rate to the decoder as it adds complexity without a clear empirical improvement; that would also cause further problems when adding dynamic refreshrate support.

**Pull request merged**